extension/demo/src/modules/storage.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+export async function set (data) {
+  console.log(data)
+  await chrome.storage.local.set(data, console.log)
+
+  return !chrome.runtime.lastError
+}
+
+export async function get (key) {
+  return new Promise(resolve => {
+    chrome.storage.local.get([key], results => {
+      console.log(results[key])
+      resolve(results[key])
+    })
+  })
+}
+
+export async function remove (key) {
+  await chrome.storage.local.remove(key)
+
+  return !chrome.runtime.lastError
+}
```

Comment:
Should use `browser` instead of `chrome` for all of these. Also make sure to put `import browser from 'webextension-polyfill'` at the top.